staging/src/k8s.io/apiserver/pkg/server/options/server_run_options_test.go

```diff
@@ -34,8 +34,8 @@ import (
 func TestServerRunOptionsValidate(t *testing.T) {
 	testRegistry := featuregate.NewComponentGlobalsRegistry()
 	featureGate := utilfeature.DefaultFeatureGate.DeepCopy()
-	effectiveVersion := utilversion.NewEffectiveVersion("1.30")
-	effectiveVersion.SetEmulationVersion(version.MajorMinor(1, 32))
+	effectiveVersion := utilversion.NewEffectiveVersion("1.32")
```

Comment:
Like Ben suggested, we can subtract the real maximum and clamp it based on the lowest version to every support this (1.31), so max(binaryVersion.SubtractMinor(MAX_SKEW), v1.31.0), and set the test case to test for binary version 1.34, so that we don't need to change it going forward.